Cargo.lock

```diff
@@ -558,7 +558,6 @@ dependencies = [
  "ethstore 0.2.0",
  "evm 0.1.0",
  "fake-hardware-wallet 0.0.1",
- "fetch 0.1.0",
```

Comment:
I think this one pops up when `git submodule init`/`git submodule update` runs. Wasm-tests something something?

Reply:
It was removed in #9020.